data/statuses.js

```diff
@@ -718,6 +718,14 @@ let BattleStatuses = {
 		num: 0,
 		duration: 3,
 		onStart(pokemon) {
+			if (pokemon.species === 'Eternatus-Eternamax') { // Special for Eternatus' Eternamax forme
+				this.add('-start', pokemon, 'Eternamax');
+				pokemon.maxhp = Math.floor(pokemon.maxhp * 2); // Assuming Eternamax is max level
+				pokemon.hp = Math.floor(pokemon.hp * 2);
+				this.add('-heal', pokemon, pokemon.getHealth, '[silent]');
+				this.effectData.duration = 0;
+				return;
+			}
```

Comment:
This can be simplified alot if we even want to do this. Would be easy enough to just so something like
if (isEternamax) {
  add start message
  set duration to 0
} else {
  standard start message
}
rest of dynamax code which you have duplicated.